test/agent_util_linux.go

```diff
@@ -107,12 +112,13 @@ func RunShellScript(path string, args ...string) error {
 	return nil
 }
 
-func RunCommand(cmd string) {
+func RunCommand(cmd string) (ouput string) {
```

Comment:
I recommend just:
```func RunCommand(cmd string) string {```
Since you do not use the variable `ouput` within the function.